jetpack-io/devbox
# [CLI] Quiet mode for all commands

## Summary
- Did a sweep for `fmt.print` and changed all instances of to to `fmt.Fprint` except in `internal/debug/debug.go`.
- Removed custom IO writer and used cobra command's `cmd.SetErr(io.Discard)` at root level.
- Changed all os.stdout writers passed to commands to `cmd.ErrOrStderr()` due to discussion around writing outputs to stderr rather than stdout.

## How was it tested?
- Compile
- run devbox commands with `-q` and without `-q`
- confirm `-q` silences outputs
